File: src/main/java/org/jukeboxmc/potion/WaterBreathingEffect.java
package org.jukeboxmc.potion;

import org.jetbrains.annotations.NotNull;
import org.jukeboxmc.entity.EntityLiving;

import java.awt.*;

/**
 * @author LucGamesYT
 * @version 1.0
 */
public class WaterBreathingEffect extends Effect {

    @Override
    public int getId() {
        return 13;
    }

    @Override
    public @NotNull EffectType getEffectType() {
        return EffectType.WATER_BREATHING;
    }

    @Override
    public @NotNull Color getEffectColor() {
        return new Color( 46, 82, 153 );
    }

    @Override
    public void apply( EntityLiving entityLiving ) {

    }

    @Override
    public void update( long currentTick ) {

    }

    @Override
    public void remove( EntityLiving entityLiving ) {

    }
}

File: JukeboxMC-Server/src/main/kotlin/org/jukeboxmc/server/effect/WaterBreathingEffect.kt
package org.jukeboxmc.server.effect

import org.jukeboxmc.api.effect.EffectType
import org.jukeboxmc.server.entity.JukeboxEntityLiving
import java.awt.Color

class WaterBreathingEffect : JukeboxEffect() {

    override fun getId(): Int {
        return 13
    }

    override fun getEffectType(): EffectType {
        return EffectType.WATER_BREATHING
    }

    override fun getEffectColor(): Color {
        return Color(46, 82, 153)
    }
    override fun apply(entityLiving: JukeboxEntityLiving) {

    }

    override fun update(currentTick: Long) {

    }

    override fun remove(entityLiving: JukeboxEntityLiving) {
    }
}